helm/node-operator-chart/templates/rbac.yaml

```diff
@@ -9,32 +9,12 @@ rules:
       - customresourcedefinitions
     verbs:
       - "*"
-  - apiGroups:
-      - extensions
-    resources:
-      - thirdpartyresources
-    verbs:
-      - "*"
-  - apiGroups:
-      - extensions
-    resources:
-      - deployments
-      - ingresses
-    verbs:
-      - "*"
   - apiGroups:
       - core.giantswarm.io
     resources:
-      - storageconfigs
```

Comment:
The node-operator does not access storageconfigs. Same applies for the kvm-operator where this got originally copied from.

Reply:
I will review it